proto/profile.proto

```diff
@@ -228,3 +233,9 @@ message Function {
   // Line number in source file.
   int64 start_line = 5;
 }
+
+message ThreadInfo {
```

Comment:
We will want to have PID / TID / Span in a lot of cases. Though this seems very tailored to our use case.